skfp/preprocessing/filters/ghose.py

```diff
@@ -0,0 +1,95 @@
+from typing import Optional
+
+from rdkit.Chem import Mol
+from rdkit.Chem.Crippen import MolLogP, MolMR
+from rdkit.Chem.Descriptors import MolWt
+from rdkit.Chem.rdMolDescriptors import CalcNumAtoms
+
+from skfp.bases.base_filter import BaseFilter
+
+
+class GhoseFilter(BaseFilter):
+    """
+    Ghose rule.
```

Comment:
Maybe `filter` instead of `rule`? After all, filter is composed from individual rules